examples/post_training_quantization/torch/ssd300_vgg16/main.py

```diff
@@ -158,11 +157,11 @@ def main():
 
     fp32_onnx_path = f"{ROOT}/ssd300_vgg16_fp32.onnx"
     torch.onnx.export(model.cpu(), dummy_input, fp32_onnx_path)
-    ov_model = mo.convert_model(fp32_onnx_path)
+    ov_model = ov.convert_model(fp32_onnx_path)
```

Comment:
Why do we still use the `PyTorch->ONNX->IR` conversion instead of the `PyTorch->IR` conversion as in `examples/post_training_quantization/torch/mobilenet_v2/main.py`?